demo/web/src/main/notebooks/A. Real-time table ops/A3 Do time series and relational joins.md

```diff
@@ -0,0 +1,49 @@
+# Time series and relational joins
+
+In our previous notebook, we showed how to filter and decorate our time tables. In this notebook, we show how to perform joins with our time series data.
```

Comment:
can we link to the notebook? soon I think

Reply:
Probably, I'll add the link and hope it works